src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.rst

```diff
@@ -75,15 +75,17 @@ where :math:`\mathbf{\bar{u}}_{COB} = [\mathrm{cob}_x, \mathrm{cob}_y, 1]^T` wit
 brightmess :math:`\mathrm{cob}_x` and :math:`\mathrm{cob}_y`, :math:`[K]` is the camera calibration matrix and
 :math:`\mathbf{r}_{COB}^N` is the unit vector describing the physical heading to the target in the camera frame.
 
-The covariance is found using the number of detected pixels and the camera parameters, given by:
+The covariance of the COB error is found using the number of detected pixels and the camera parameters, given by:
 
 .. math::
 
-    P = \frac{2 \ pi}{\sqrt{\mathrm{numPixels}}} \left( \begin{bmatrix} d_x^2 & 0 & 0 \\ 0 & d_y^2 & 0
+    P = \frac{\sqrt{\mathrm{numPixels}}}{2 \pi} \left( \begin{bmatrix} d_x^2 & 0 & 0 \\ 0 & d_y^2 & 0
```

Comment:
Is this 4/pi divided by the norm?

Reply:
Yes, good catch. I forgot to update the documentation after we changed that computation again.